osu.Game/Beatmaps/ControlPoints/ControlPoint.cs

```diff
@@ -25,6 +25,14 @@ public abstract class ControlPoint : IComparable<ControlPoint>, IEquatable<Contr
         /// <returns>Whether equivalent.</returns>
         public abstract bool EquivalentTo(ControlPoint other);
 
+        /// <summary>
+        /// Whether this control point results in a meaningful change when placed after another.
+        /// </summary>
+        /// <param name="other">Another control point to compare with.</param>
+        /// <param name="time">The time this timing point will be placed at.</param>
```

Comment:
do you mean "other" when you say "this"? If so, reference the param.

Also timing point -> control point.

Reply:
It's intended to be called from the new timing point, with other being the already existing timing point. I think I'll rename the parameter so that this is more clear. (existing instead of just other)
This is significant since only existing timing points will have a proper Time value to compare to.

Will change timing point to control point.